projects/laji/src/app/+taxonomy/taxon/info-card/taxon-identification/taxon-identification.component.ts

```diff
@@ -49,12 +57,93 @@ export class TaxonIdentificationComponent implements OnChanges, AfterViewInit, O
     viewChange: this.infiniteScrollStatusCheck$.pipe(
       filter(() => this.loadMoreElem && this.isWithinXPixelsOfViewport(this.loadMoreElem.nativeElement, INFINITE_SCROLL_DISTANCE)),
       map(() => ({
-          start: 0,
-          end: this.children.length
-        }))
+        start: 0,
+        end: this.children.length
+      }))
     )
   };
 
+  private parseTaxonDescriptions(child: Taxonomy) {
+    if (!child.descriptions || child.descriptions.length < 1) { return undefined; }
+
+    const descriptions = child.descriptions;
+    const taxonDescriptions = {};
+
+    const requestedVariables = {
+      'MX.SDVG1': [
+        'MX.descriptionText',
+        'MX.identificationText',
+        'MX.descriptionMicroscopicIdentification'
+      ],
+      'MX.SDVG2': [
+        'MX.distributionFinland'
+      ],
+      'MX.SDVG4': [
+        'MX.reproductionFloweringTime'
+      ],
+      'MX.SDVG5': [
+        'MX.habitat',
+        'MX.habitatSubstrate'
+      ],
+      'MX.SDVG8': [
+        'MX.growthFormAndGrowthHabit',
+        'MX.descriptionOrganismSize',
+        'MX.descriptionStem',
+        'MX.descriptionLeaf',
+        'MX.descriptionRoot',
+        'MX.descriptionFlower',
+        'MX.descriptionFruitAndSeed',
+        'MX.descriptionCone',
+        'MX.descriptionThallus',
+        'MX.descriptionFruitbody',
+        'MX.descriptionSpore',
+        'MX.descriptionSporangiumAndAsexualReproduction',
+        'MX.algalPartnerOfLichen'
+      ],
+    };
+
+    descriptions.forEach(description => {
+      description.groups.forEach(group => {
+        if (Object.keys(requestedVariables).includes(group.group)) {
+          group.variables.forEach(variable => {
+            if (requestedVariables[group.group].includes(variable.variable) && !taxonDescriptions[variable.variable]) {
+              const title = variable.title;
+              const content = variable.content;
+              taxonDescriptions[variable.variable] = { title, content };
+
+              if (description.title && !this.descriptionSources.includes('<p>' + description.title + '</p>')) {
```

Comment:
There's no reason to put the `<p>` tags in `descriptionSources`. Let the html worry about how stuff is rendered and only provide the data in javascript (ie. avoid html in variables whenever you can avoid it).